internal/helm/repository/utils.go

```diff
@@ -35,3 +47,19 @@ func NormalizeURL(repositoryURL string) string {
 	return strings.TrimRight(repositoryURL, "/") + "/"
 
 }
+
+// ValidateDepURL returns an error if the given depended repository URL declaration is not supported
+// The reason for this is that the dependency manager will not be able to resolve the alias declaration
+// e.g. repository: "@fantastic-charts"
+func ValidateDepURL(repositoryURL string) error {
+	switch {
+	case strings.Contains(repositoryURL, helmreg.OCIScheme):
+		return nil
+	case strings.Contains(repositoryURL, "https://") || strings.Contains(repositoryURL, "http://"):
+		return nil
+	case strings.Contains(repositoryURL, alias):
+		return fmt.Errorf("%w: %s", errInvalidAliasedDep, repositoryURL)
+	default:
+		return fmt.Errorf("%w: %s", errInvalidDepURL, repositoryURL)
```

Comment:
```suggestion
	case strings.HasPrefix(repositoryURL, helmreg.OCIScheme):
		return nil
	case strings.HasPrefix(repositoryURL, "https://") || strings.Contains(repositoryURL, "http://"):
		return nil
	case strings.HasPrefix(repositoryURL, alias):
		return fmt.Errorf("%w: %s", errInvalidAliasedDep, repositoryURL)
	default:
		return fmt.Errorf("%w: %s", errInvalidDepURL, repositoryURL)
```

This is safer I think, as technically `@` is also a valid symbol in a URL.